server/migrations/sql/20191025170913-create-scenarios-to-tags.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE scenarios_to_tags(
+    id SERIAL PRIMARY KEY,
```

Comment:
dont need an `id` here since we can set the PRIMARY KEY to `(scenario_id, tag_id)`